src/commands/geohash.json

```diff
@@ -4,7 +4,7 @@
         "complexity": "O(log(N)) for each member requested, where N is the number of elements in the sorted set.",
         "group": "geo",
         "since": "3.2.0",
-        "arity": -2,
+        "arity": -3,
```

Comment:
the docs are generated by this same json file, so one could argue that the problem is the fact that the member argument isn't marked as optional, not the wrong arity.

of course it's silly to call this command with 0 members (it just returns an empty array), but maybe some application or library is wired to call the command on whatever input array, in in some cases the caller passes an empty array.
in the past it used to succeed and now it'll error.

so although it's idiotic, maybe we should take the safe path ans keep supporting it?
@yossigo WDYT?

Reply:
@oranagra I agree, I don't think we should change this.

Reply:
@hwware please revert the arity change, and instead add an `optional` flag to `member` argument.
thanks.